apps/publish.go

```diff
@@ -273,6 +280,43 @@ func publishApp(ctx context.Context, cfg *config.Config, token string, app *App,
 	return a.AppMarketplaceEntry, nil
 }
 
+func installCookies(ctx context.Context, cfg *config.Config, token string, manifestInputs ManifestInputs, app *App) error {
+	createCookieURL := fmt.Sprintf("organizations/%s/cookies", app.OrgCode)
```

Comment:
why does this not create an app like everything else?

Reply:
replaced by creating an app with cookies field

Reply:
So instead of user installing an app to get cookies template it happens directly on app publish? That org code is the org that developed the app